apps/api/README.md

```
@@ -1,3 +1,30 @@
-# API
+# Marble API
 
-API endpoints users can fetch data from.
+Prerequisites:
+
+- [Vercel CLI](https://vercel.com/docs/cli) installed globally
+
+To develop locally:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)